script.js

```diff
@@ -0,0 +1,89 @@
+let potionDecline = "You declined sorceress offer of strength potion"
+let villagePeace = "You decided to find peaceful solution with the rogue leader"
+let dragonCompromise = "You decided to find a compromise with a dragon instead of a fight with him"
+let dragonFight = "You decided to fight with the dragon and died but become a legend"
+let mistake = "you did a mistake, try again"
+
+alert("Welcome onboard in our time-machine lab... You will play for Luke Skywalker from medieval times. You would like to become a knight and here is your story")
+alert("You serve to Sir Robert, and to become a real knight you have to pass through all competition that Sir Robert will provide you or request to do from you")
+let firstDaySelect = prompt("Decide your first day adventure. Input 'forest' , 'village' or 'castle' to choose your next location")
+
+if (firstDaySelect === "forest") {
+    alert("On the first day you have received a message that you have to go to the forest. You have met a sorceress on your way. She is offering you a strength potion for promise.")
+    let potionDecision = confirm("Would you like to accept her offer?")
+    if (potionDecision === false) {
+        firstDaySelect = potionDecline
+    }
```

Comment:
could be written as:
```suggestion
    if (!potionDecision) {
        firstDaySelect = potionDecline
    }
```